src/lib/marquee.scss

```diff
@@ -1,32 +1,32 @@
-.marquee,
 .marquee__slide {
-  display: grid;
-  grid-auto-flow: column;
+  display: inline-block;
   width: max-content;
 }
-
 .marquee {
-  animation: scroll var(--duration) linear infinite var(--animation-state, running);
-
   /*
-  * On Firefox, there is an issue where the Marquee does not play unless
-  * the window is resized or the play state is toggled.
+  * On Firefox & Safari, there was an issue where the Marquee does not play
+  * unless the window is resized or the play state is toggled.
   *
-  * Setting an animation delay is a workaround fix for this issue.
-  * A low delay of 0.01s is set so it does not interfere with the animation.
+  * This was happening because the animation was loaded before the marqueeWidth
```

Comment:
👏 for adding the description of the issue @chanroyc 

reading it actually jogged my memory and had me go 'oh yeah that seems familiar' but mb for not remembering to begin with 😭